extensions/amp-slike/0.1/test/validator-amp-slike.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
```

Comment:
Needs a copyright comment header and documentation. See similar `.html` test files in other extension directories.